Jenkinsfile

```diff
@@ -992,6 +996,12 @@ def lint() {
     """
 }
 
+def gn_check() {
+    sh """
+        gn check ${OUT_DIR}
```

Comment:
isn’t build dir available only after build?

Reply:
do we want this added to Windows as well?

Reply:
sort of - the dir is generated after `gn gen out/Release`. We don't have to compile everything to do `gn check`

Reply:
@iefremov would the directory exist after `npm run init`? I don't think so, right? Is it safe to make an empty directory? or does it expect content to be there. Confused why it needs the directory if compilation not needed

Reply:
sorry for the confusion. The checker needs your "setup" to be configured - i.e. all variables in gn files to be set correctly. To do this, you have to run `gn gen out/<...>` for the build configuration that you want. I suppose that in our brave world `gn gen` happens as part of `npm run build`